x-pack/test/fleet_api_integration/apis/outputs/crud.ts

```diff
@@ -1352,29 +1350,6 @@ export default function (providerContext: FtrProviderContext) {
         });
       });
 
-      it('should discard the shipper values when shipper is disabled', async function () {
-        await supertest
-          .post(`/api/fleet/outputs`)
-          .set('kbn-xsrf', 'xxxx')
-          .send({
-            name: 'default monitoring output 1',
-            type: 'elasticsearch',
-            hosts: ['https://test.fr'],
-            is_default_monitoring: true,
-            shipper: {
-              disk_queue_enabled: true,
-              disk_queue_path: 'path/to/disk/queue',
-              disk_queue_encryption_enabled: true,
-            },
-          })
-          .expect(200);
-        const {
-          body: { items: outputs },
-        } = await supertest.get(`/api/fleet/outputs`).expect(200);
-        const defaultOutputs = outputs.filter((o: any) => o.is_default_monitoring);
-        expect(defaultOutputs[0].shipper).to.equal(null);
-      });
-
```

Comment:
Duplicate of https://github.com/elastic/kibana/blob/ffe6cf4de75d412dbd1d501e01cd8cae05597bfa/x-pack/test/fleet_api_integration/apis/outputs/crud.ts#L1290-L1311 in the same file